--- near/intents-docs/api-reference__oneclick__submit-a-signed-intent__484a1a5106:previous
+++ near/intents-docs/api-reference__oneclick__submit-a-signed-intent__484a1a5106:current
@@ -10,13 +10,23 @@
 submit-intent
 Try it
 Submit a signed intent
 cURL
-curl --request POST \
---url https://1click.chaindefuser.com/v0/submit-intent \
---header 'Content-Type: application/json' \
---header 'X-API-Key: <api-key>' \
---data '
+curl
+--request
+POST
+\
+--url
+https://1click.chaindefuser.com/v0/submit-intent
+\
+--header
+'Content-Type: application/json'
+\
+--header
+'X-API-Key: <api-key>'
+\
+--data
+'
 {
 "type": "swap_transfer",
 "signedData": {
 "payload": {
@@ -146,10 +156,15 @@
 puts response.read_body
 200
 400
 {
-"intentHash": "44XpLRAuZKoVGs9T4qbSNv33MDKMePPAibA52geVLWFw",
-"correlationId": "550e8400-e29b-41d4-a716-446655440000"
+"intentHash"
+:
+"44XpLRAuZKoVGs9T4qbSNv33MDKMePPAibA52geVLWFw"
+,
+"correlationId"
+:
+"550e8400-e29b-41d4-a716-446655440000"
 }
 {
 "message": "error message"
 }
@@ -215,13 +230,23 @@
 ⌘
 I
 Submit a signed intent
 cURL
-curl --request POST \
---url https://1click.chaindefuser.com/v0/submit-intent \
---header 'Content-Type: application/json' \
---header 'X-API-Key: <api-key>' \
---data '
+curl
+--request
+POST
+\
+--url
+https://1click.chaindefuser.com/v0/submit-intent
+\
+--header
+'Content-Type: application/json'
+\
+--header
+'X-API-Key: <api-key>'
+\
+--data
+'
 {
 "type": "swap_transfer",
 "signedData": {
 "payload": {
@@ -351,10 +376,15 @@
 puts response.read_body
 200
 400
 {
-"intentHash": "44XpLRAuZKoVGs9T4qbSNv33MDKMePPAibA52geVLWFw",
-"correlationId": "550e8400-e29b-41d4-a716-446655440000"
+"intentHash"
+:
+"44XpLRAuZKoVGs9T4qbSNv33MDKMePPAibA52geVLWFw"
+,
+"correlationId"
+:
+"550e8400-e29b-41d4-a716-446655440000"
 }
 {
 "message": "error message"
 }
