--- hpl/direct/hyperliquid-portfolio-margin:previous
+++ hpl/direct/hyperliquid-portfolio-margin:current
@@ -49,9 +49,9 @@
 Account abstraction modes
 Next
 Margin tiers
 Last updated
-3 days ago
+4 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/portfolio-margin.md
 LINK | Next Margin tiers | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/margin-tiers
 LINK | Previous Account abstraction modes | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/account-abstraction-modes
