--- near/direct/near-home:previous
+++ near/direct/near-home:current
@@ -85,30 +85,30 @@
 LINK | NEAR Confidential Onchain doesn’t have to mean public. The technology that keeps your balances and activity confidential — while you control who sees what. | https://x.com/NEARProtocol/status/2074515816670781812?s=20
 LINK | NEAR Intents The engine behind every swap. Market makers compete to get you the best price across 35+ networks — settled in seconds. | https://docs.near-intents.org
 LINK | NEAR Protocol One of the fastest, most reliable blockchains in the world. The foundation that keeps your account secure and your transactions instant. | https://near.org
 [[FRONTEND SCRIPT ASSETS]]
-SCRIPT | https://near.com/_next/static/chunks/0.7vofpba8mq3.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/03~yq9q893hmn.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/04tjshshq8gdu.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/065dlhdy2db3j.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/06_e-26tt4o~a.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/06tkxowir-vxv.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/07mc.hk0nx._2.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/08~qplg140~wg.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/093gridn94m7i.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0_zp8-7ss~zod.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0aldwhtm.olqc.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0bjfd.v4zz1z6.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0dhr~lg7u.jag.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0ekd0elz1p199.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0m04-nr1tfy-b.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0oe-ieqhtb639.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0uz~0go0s7d~r.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0vl.5i0i6..f7.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0z-38g9i3f8x4.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/0~a5gg6nl.po..js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/106w1ocd8yls2.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/10lzq.77~o6pa.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/121ra9um8~sdf.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/15m9dnzl223rm.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/15n71wxukuoyu.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
-SCRIPT | https://near.com/_next/static/chunks/turbopack-08-kn3ie5o9b7.js?dpl=dpl_BL3QmVuCSpLJSfQCAoWpgXTsDEdN
+SCRIPT | https://near.com/_next/static/chunks/0.7vofpba8mq3.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/03~yq9q893hmn.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/04tjshshq8gdu.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/065dlhdy2db3j.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/06_e-26tt4o~a.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/06tkxowir-vxv.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/07mc.hk0nx._2.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/08~qplg140~wg.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/093gridn94m7i.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0_zp8-7ss~zod.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0aldwhtm.olqc.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0bjfd.v4zz1z6.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0dhr~lg7u.jag.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0ekd0elz1p199.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0m04-nr1tfy-b.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0oe-ieqhtb639.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0uz~0go0s7d~r.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0vl.5i0i6..f7.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0z-38g9i3f8x4.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/0~a5gg6nl.po..js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/106w1ocd8yls2.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/10lzq.77~o6pa.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/121ra9um8~sdf.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/15m9dnzl223rm.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/15n71wxukuoyu.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
+SCRIPT | https://near.com/_next/static/chunks/turbopack-08-kn3ie5o9b7.js?dpl=dpl_vCkFd6Lp4WLReaM1U3UNgLrYf7Hs
