--- near/rhea-guide/developers__contracts__c832df276a:previous
+++ near/rhea-guide/developers__contracts__c832df276a:current
@@ -0,0 +1,228 @@
+For the complete documentation index, see
+llms.txt
+. This page is also available as
+Markdown
+.
+All contracts are on the NEAR Protocol. The source code of RHEA Finance can be found on
+Github
+.
+Find below the list of all accounts that have directly managed or currently manage the affairs of RHEA Finance.
+Mainnet Contracts
+Address
+Type
+Mission
+Locked?
+Owner
+Source Code
+v2.ref-finance.near
+Exchange Contract
+Manage the Automated Market Maker functions; Swap and Provide Liquidity
+Yes
+ref-finance.sputnik-dao.near
+Link
+boostfarm.ref-labs.near
+Farming Contract
+Manage liquidity incentives
+Yes
+ref-finance.sputnik-dao.near
+Link
+ref-ve.ref-labs.near
+Vetokenomics Contract
+Manage Vote-Escrowed Token (veToken)
+No (In Progress)
+ref-dev-team.near
+Link
+xtoken.ref-finance.near
+Staking Contract
+Mint and burn xREF, and Distribute time-based rewards
+Yes
+ref-finance.sputnik-dao.near
+Link
+dclv2.ref-labs.near
+Concentrated Liquidity Contract (Ref v2)
+Manage concentrated liquidity and limit order functions
+No (In Progress)
+ref-dev-team.near
+Link
+ref-finance.sputnik-dao.near
+Sputnik DAO Contract
+Ensure the success of Ref by taking strategic decisions (incl. smart contract amendments)
+Yes
+Multisig (
+Link
+)
+Link
+ref-community-board.sputnik-dao.near
+Sputnik DAO Contract
+Manage and allocate funds to specific community contributors
+Yes
+Multisig (
+Link
+)
+Link
+dao.ref-dev-team.near
+Sputnik DAO Contract
+Execute the Strategy and Roadmap
+Yes
+Multisig (
+Link
+)
+Link
+ref.ref-dev-fund.near
+Vesting Contract
+Manage REF vesting contracts of Dev DAO members
+Yes
+dao.ref-dev-team.near
+Link
+s01.ref-airdrop.near
+Airdrop Contract
+Manage first REF airdrop
+Yes
+N/A
+Link
+ref-bug-bounty-1.near
+Simple Address
+Manage one-time bug bounty payments
+N/A
+token.v2.ref-finance.near
+$REF Fungible Token Contract
+Mint REF token
+Yes
+N/A
+Link
+ref-finance.near
+Exchange Contract
+v1 (obsolete) - Manage the Automated Market Maker functions; Swap and Provide Liquidity
+No
+N/A
+Link
+token.ref-finance.near
+Fungible Token Contract
+v1 (obsolete) - Mint REF token
+Yes
+N/A
+v2.ref-farming.near
+Farming Contract
+v2 (obsolete) - Manage liquidity incentives
+Yes
+ref-finance.sputnik-dao.near
+Link
+dcl.ref-labs.near
+Concentrated Liquidity Contract (Ref v2)
+Beta (obsolete) - Manage concentrated liquidity and limit order functions
+No
+ref-dev-team.near
+Link
+refchef.near
+Simple Address
+Manage inter-account transactions
+N/A
+ref-farm-reward-proxy.near
+Simple Address
+Manage third-party deposits for liquidity incentives
+N/A
+a4b55d572d5f41c0dbbb88fd1317ebff87edcc75cad5d8705c55e94c48993926
+Simple Address
+Manage Protocol Revenue Conversion - REF buyback (Q2 2022)
+N/A
+28d262719c5d97e3c570a5a71a817820300a97bd086136923e44812193ef6c4d
+Simple Address
+Manage Protocol Revenue Conversion - REF buyback (Q3 2022)
+N/A
+ref-dev-teller.near
+Simple Address
+Manage inter-account transactions
+N/A
+0x102d7FaD37A4e0266A0AFDcDc90A04408F9ac091
+Simple Address (Ethereum)
+Manage TRI<>REF Liquidity Provision on Tri Solaris
+N/A
+0x45d76fa8498239ed5fc4dcd7377f92beec5c48d6
+Simple Address
+(Ethereum)
+Manage strategic OTC
+deal
+(2022)
+N/A
+0x04dFb979Ffa1Fb08B6e6EF626377161CBB085Cf1
+Aggregator Contract-Stargate-Rainbow(Aurora)
+Manage Stargate - Rainbow Bridge Asset
+N/A
+lst.rhealab.near
+LST Contract
+Manage rNear Token
+Yes
+Testnet Contracts
+Address
+Type
+Mainnet counterpart
+ref-finance-101.testnet
+Exchange Contract
+v2.ref-finance.near
+boostfarm.ref-finance.testnet
+Farming Contract
+boostfarm.ref-labs.near
+v010.refve.testnet
+Vetokenomics Contract
+ref-ve.ref-labs.near
+xref.ref-finance.testnet
+Staking Contract
+xtoken.ref-finance.near
+dclv2.ref-dev.testnet
+Concentrated Liquidity Contract (Ref v2)
+dclv2.ref-labs.near
+ref.fakes.testnet
+$REF Fungible Token Contract
+token.v2.ref-finance.near
+Timelock
+Timelocks are a smart contract feature that states that some actions will only be performed after a certain period of time rather than immediately.
+RHEA Finance does not use/have a timelock feature. Ref contracts are directly upgraded from the DAO (ref-finance.sputnik-dao.near), thus providing time to users, via the voting period of the associated proposal, to protect their funds in case they identify suspicious activities.
+DAO proposals usually take from 48 to 72 hours either to get approved or rejected.
+Frozen List
+The RHEA Finance Exchange contract (v2.ref-finance.near) has a 'Frozen List' feature, which was deployed on version 1.6.0+.
+The feature enables the owner and/or
+Guardians
+of the contract to freeze any token listed on the exchange. When frozen, any action (Swap, Add/Remove liquidity, Deposit, Withdraw, etc.) on the corresponding token will fail, resulting in a 'panic error' at the contract level.
+The owner and/or
+Guardians
+can use the following commands to freeze/unfreeze any token:
+To get the Frozen List, anyone can query the following command:
+Motivation
+The Frozen List feature has been designed to mitigate the impact of any attack or hack associated to a specific token and its potential contagion within the ecosystem.
+The feature can also be activated as a preventive measure. For example, in November 2022, Metapool identified a critical
+bug
+on its contract. RHEA Finance was informed and immediately froze the STNEAR token, preventing users from any associated action (trading and pooling) on Ref, thus reducing the severity of the impact (trading losses, extreme volatility, etc.) that such a bug could have caused to traders and LPs.
+Previous
+History
+Next
+Guardians
+Last updated
+1 year ago
+Was this helpful?
+[[LINK TARGETS]]
+LINK | Github | https://github.com/ref-finance
+LINK | Guardians | https://guide.rhea.finance/developers/guardians
+LINK | Link | https://app.astrodao.com/dao/ref-community-board.sputnik-dao.near/groups/all
+LINK | Link | https://app.astrodao.com/dao/ref-finance.sputnik-dao.near/groups/all
+LINK | Link | https://dev-fund.ref-finance.com/#/dao.ref-dev-team.near/
+LINK | Link | https://github.com/near-daos/sputnik-dao-contract
+LINK | Link | https://github.com/ref-finance/boost-farm
+LINK | Link | https://github.com/ref-finance/ref-contracts/tree/main/ref-exchange
+LINK | Link | https://github.com/ref-finance/ref-contracts/tree/main/ref-farming
+LINK | Link | https://github.com/ref-finance/ref-dcl
+LINK | Link | https://github.com/ref-finance/ref-dev-fund/tree/session_vault/session_vault
+LINK | Link | https://github.com/ref-finance/ref-token/tree/deployed-ref-token/ref-token
+LINK | Link | https://github.com/ref-finance/ref-token/tree/master/xref-token
+LINK | Link | https://github.com/ref-finance/ref-ve
+LINK | Link | https://github.com/skyward-finance/contracts/tree/master/lockup
+LINK | Markdown | https://guide.rhea.finance/developers/contracts.md
+LINK | Next Guardians | https://guide.rhea.finance/developers/guardians
+LINK | Previous History | https://guide.rhea.finance/roadmap-1
+LINK | [no text] | https://guide.rhea.finance/developers/contracts#frozen-list
+LINK | [no text] | https://guide.rhea.finance/developers/contracts#mainnet-contracts
+LINK | [no text] | https://guide.rhea.finance/developers/contracts#motivation
+LINK | [no text] | https://guide.rhea.finance/developers/contracts#testnet-contracts
+LINK | [no text] | https://guide.rhea.finance/developers/contracts#timelock
+LINK | bug | https://twitter.com/meta_pool/status/1588245211850776577?s=20&t=K-osl2RR5sjMkHBfoVG9gg
+LINK | deal | https://gov.ref.finance/t/a-strategic-ref-otc-wip/448
+LINK | llms.txt | https://guide.rhea.finance/llms.txt
