--- near/rhea-guide/near-chain-guides__aggregator-bridge__323cac4af1:previous
+++ near/rhea-guide/near-chain-guides__aggregator-bridge__323cac4af1:current
@@ -1,36 +1,17 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
+> For the complete documentation index, see [llms.txt](https://guide.rhea.finance/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://guide.rhea.finance/near-chain-guides/aggregator-bridge.md).
+# Aggregator Bridge
 How to bridge?
-Step1: Click on the "Bridge" tab and choose Aggregate Bridge
-Step2: Select the chain you want to bridge "from" and "to" for the asset you want to bridge.
-Step3: Connect the wallet of the chain you selected.
-Step4: Enter the amount of the assets you want to bridge.
+#### Step1: Click on the "Bridge" tab and choose Aggregate Bridge&#x20;
+<figure><img src="/files/7pLPyqBs2GWuL5vPJhAM" alt=""><figcaption></figcaption></figure>
+#### Step2: Select the chain you want to bridge "from" and "to" for the asset you want to bridge.&#x20;
+<figure><img src="/files/UMufRTcBcCYGTQiFPi9c" alt="" width="563"><figcaption></figcaption></figure>
+#### Step3: Connect the wallet of the chain you selected.
+<figure><img src="/files/u4VIU6USi3IeXQNSadao" alt=""><figcaption></figcaption></figure>
+#### Step4: Enter the amount of the assets you want to bridge.&#x20;
 The aggregator will automatically calculate and choose the best amount out.
-Step 5: Click "
-Preview"
-to view the bridge preview.
-Step 6: Click "
-Transfer"
-to sign transaction.
-You can check the progress anytime through the "Bridge Transaction History" page
-Previous
-NEAR Liquid Staking Tokens (LST)
-Next
-RHEA Point System
-Last updated
-7 months ago
-Was this helpful?
-[[LINK TARGETS]]
-LINK | Markdown | https://guide.rhea.finance/near-chain-guides/aggregator-bridge.md
-LINK | Next RHEA Point System | https://guide.rhea.finance/near-chain-guides/rhea-point-system
-LINK | Previous NEAR Liquid Staking Tokens (LST) | https://guide.rhea.finance/near-chain-guides/near-liquid-staking-tokens-lst
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step-5-click-preview-to-view-the-bridge-preview
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step-6-click-transfer-to-sign-transaction.-you-can-check-the-progress-anytime-through-the-bridge-tra
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step1-click-on-the-bridge-tab-and-choose-aggregate-bridge
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step2-select-the-chain-you-want-to-bridge-from-and-to-for-the-asset-you-want-to-bridge
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step3-connect-the-wallet-of-the-chain-you-selected
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/aggregator-bridge#step4-enter-the-amount-of-the-assets-you-want-to-bridge
-LINK | llms.txt | https://guide.rhea.finance/llms.txt
+<figure><img src="/files/NKC86I7HSljtbNm3L0rM" alt=""><figcaption></figcaption></figure>
+#### Step 5: Click "**Preview"** to view the bridge preview.
+<figure><img src="/files/4Ns97nsCl4aoSx7QgMfo" alt="" width="563"><figcaption></figcaption></figure>
+#### Step 6: Click "**Transfer"** to sign transaction. You can check the progress anytime through the "Bridge Transaction History" page
+<figure><img src="/files/hDuTPqlCl69XJeEwCLy0" alt="" width="563"><figcaption></figcaption></figure>
+<figure><img src="/files/D2GTdopRKPDO5LEEUZP4" alt=""><figcaption></figcaption></figure>
