--- near/rhea-guide/near-chain-guides__lending-and-borrowing__lp-as-collateral__512a27e58b:previous
+++ near/rhea-guide/near-chain-guides__lending-and-borrowing__lp-as-collateral__512a27e58b:current
@@ -1,28 +1,11 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
+> For the complete documentation index, see [llms.txt](https://guide.rhea.finance/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/lp-as-collateral.md).
+# LP as Collateral
 Increase the capital efficiency of LP assets!
 LP tokens are no longer limited to farming, they can also be supplied to Burrow as collateral to borrow other assets at the same time.
-Use Case
+## **Use Case** <a href="#use-case" id="use-case"></a>
 When capital is tight and you want to both retain LP income and borrow other assets, you can use LP as collateral.
-LP Price
+## LP Price <a href="#lp-price" id="lp-price"></a>
 The price of LP is ensured by Ref's contract. The Burrow contract will automatically fetch the price from the Ref contract during on-chain operations to ensure real-time accuracy.
-Separate position
+## Separate position <a href="#separate-position" id="separate-position"></a>
 From a security perspective, LP Collateral positions are independent. Each LP token type has its own separate position. When borrowing other assets, users need to select the collateral position they wish to use.
-Previous
-Audits & Risks
-Next
-How Lending & Borrowing works
-Last updated
-1 year ago
-Was this helpful?
-[[LINK TARGETS]]
-LINK | Markdown | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/lp-as-collateral.md
-LINK | Next How Lending & Borrowing works | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/how-lending-and-borrowing-works
-LINK | Previous Audits & Risks | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/audits-and-risks
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/lp-as-collateral#lp-price
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/lp-as-collateral#separate-position
-LINK | [no text] | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/lp-as-collateral#use-case
-LINK | llms.txt | https://guide.rhea.finance/llms.txt
+\ <br>
