--- near/rhea-guide/near-chain-guides__yield-boost__xrhea-farming-boost__a6af8f2149:previous
+++ near/rhea-guide/near-chain-guides__yield-boost__xrhea-farming-boost__a6af8f2149:current
@@ -0,0 +1,170 @@
+For the complete documentation index, see
+llms.txt
+. This page is also available as
+Markdown
+.
+Overview
+xRHEA Farming Boost allows users to lock xRHEA tokens in exchange for boosted incentive rewards on their
+farm APYs
+. The longer the lock-up period, the greater the boost — up to a maximum of 6 months.
+xRHEA Farming Boost
+interface with boost ratio range and lock-up period up to 6 months
+List of 7 Boosted farm Pools on RHEA
+Farm Pools
+Link
+rNEAR-NEAR
+https://app.rhea.finance/sauce/6494
+RHEA-NEAR
+https://app.rhea.finance/pool/6458
+NEAR-ZEC
+https://app.rhea.finance/pool/6065
+NEAR-PUBLIC
+https://app.rhea.finance/pool/6503
+mpDAO - STNEAR
+https://app.rhea.finance/pool/5438
+nBTC - NEAR
+https://app.rhea.finance/sauce/5949
+DOG - NBTC
+https://app.rhea.finance/pool/6424
+How it works
+Lock xRHEA → Earn Weight → Get Boosted APYs on Farming
+The program operates in two sections:
+Weight Algorithm – Calculates your Weight based on the amount and duration of xRHEA locked.
+Farming Multiplier Algorithm – Converts your Weight into a reward boost factor.
+1. Weight Algorithm
+Your Weight is determined by:
+Where:
+Y = Weight
+X = The amount of xRHEA locked
+D = Lock duration (from 30 to 180 days)
+D_min = 30
+D_max = 180
+M_min = 10,000
+M_max = 30,000
+Example Weight Table
+xRHEA Locked
+Lock Duration
+Weight Received
+100 xRHEA
+1 Month
+100
+100 xRHEA
+3.5 Months
+200
+100 xRHEA
+6 Months
+300
+The longer you lock, the higher the Weight (up to 3x boost at 180 days)
+2. Farming Multiplier Algorithm
+Once Weight is calculated, it’s converted into a Farming Multiplier, which boosts the user’s share of the incentive rewards pool.
+Where:
+Y = user's Weight
+boost_suppress_factor = 100
+base = 38634105686174997042188779520
+Farming Multiplier Table
+Weight Amount
+Farming Multiplier
+1,500
+1.111x
+15,000
+1.205x
+150,000
+1.300x
+1,500,000
+1.394x
+15,000,000
+1.488x
+Example: How Rewards Are Distributed
+Imagine the rNEAR-NEAR farm emits 9.89% APY in incentive rewards.
+Case 1: New User Without xRHEA Lock when farming
+User
+USD value staked in farm
+Weight
+Multiplier
+APY
+Boosted APY
+Charlie
+1,000
+0
+1.000x
+9.89%
+-
+Case 2: xRHEA Lock Boosted Users when farming
+User
+USD value staked in farm
+Weight
+Multiplier
+APY
+Boosted APY
+Alice
+500
+150,000
+1.300x
+9.89%
+12.85%
+Bob
+50
+150
+1.056x
+9.89%
+10.44%
+Updated incentive rewards allocation for rNEAR-NEAR farm
+Alice: 12.85% APY
+Bob: 10.44% APY
+Charlie: 9.89% APY (Not Boosted)
+How to stake xRHEA for farming boost: Step by Step walkthrough
+Step 1: Go to
+https://app.rhea.finance/stake
+and connect your wallet
+Step 2: Click Farming Boost:
+Step 3: Choose RHEA or xRHEA:
+Stake requires minimum of 1.7 RHEA/xRHEA
+Step 4: Pick Boost Ratio Range based on your Lock time.
+Lock-up period from 1~6 month
+Step 4.1: If you hover to the Boosted ratio text, there will be a quick preview pop-up of your Boosted Farm APYs based on Stake Amount and Lock time of xRHEA
+Step 5: Click Stake and you're done!
+Step 6: you head over to the
+farms section
+on Liquidity tab, you can now see your boosted APY figures under the APYs.
+Previous
+Yield Boost
+Next
+xRHEA Lending Boost
+Last updated
+11 months ago
+Was this helpful?
+[[LINK TARGETS]]
+LINK | Markdown | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost.md
+LINK | Next xRHEA Lending Boost | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-lending-boost
+LINK | Previous Yield Boost | https://guide.rhea.finance/near-chain-guides/yield-boost
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#case-1-new-user-without-xrhea-lock-when-farming
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#case-2-xrhea-lock-boosted-users-when-farming
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#example-how-rewards-are-distributed
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#example-weight-table
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#farming-multiplier-table
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#how-it-works
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#how-to-stake-xrhea-for-farming-boost-step-by-step-walkthrough
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#id-1.-weight-algorithm
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#id-2.-farming-multiplier-algorithm
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#list-of-7-boosted-farm-pools-on-rhea
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#overview
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-1-go-to-https-app.rhea.finance-stake-and-connect-your-wallet
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-2-click-farming-boost
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-3-choose-rhea-or-xrhea
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-4-pick-boost-ratio-range-based-on-your-lock-time
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-4.1-if-you-hover-to-the-boosted-ratio-text-there-will-be-a-quick-preview-pop-up-of-your-boosted
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-5-click-stake-and-youre-done
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-6-you-head-over-to-the-farms-section-on-liquidity-tab-you-can-now-see-your-boosted-apy-figures
+LINK | [no text] | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#updated-incentive-rewards-allocation-for-rnear-near-farm
+LINK | farm APYs | https://app.rhea.finance/pools?activeTab=farm
+LINK | farms section | https://guide.rhea.finance/near-chain-guides/yield-boost/xrhea-farming-boost#step-1-go-to-https-app.rhea.finance-stake-and-connect-your-wallet
+LINK | https://app.rhea.finance/pool/5438 | https://app.rhea.finance/pool/5438
+LINK | https://app.rhea.finance/pool/6065 | https://app.rhea.finance/pool/6065
+LINK | https://app.rhea.finance/pool/6424 | https://app.rhea.finance/pool/6424
+LINK | https://app.rhea.finance/pool/6458 | https://app.rhea.finance/pool/6458
+LINK | https://app.rhea.finance/pool/6503 | https://app.rhea.finance/pool/6503
+LINK | https://app.rhea.finance/sauce/5949 | https://app.rhea.finance/sauce/5949
+LINK | https://app.rhea.finance/sauce/6494 | https://app.rhea.finance/sauce/6494
+LINK | https://app.rhea.finance/stake | https://app.rhea.finance/stake
+LINK | llms.txt | https://guide.rhea.finance/llms.txt
+LINK | xRHEA Farming Boost | https://app.rhea.finance/stake
