--- near/rhea-guide/near-chain-guides__lending-and-borrowing__06b8e77db5:previous
+++ near/rhea-guide/near-chain-guides__lending-and-borrowing__06b8e77db5:current
@@ -1,24 +1,7 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
+> For the complete documentation index, see [llms.txt](https://guide.rhea.finance/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://guide.rhea.finance/near-chain-guides/lending-and-borrowing.md).
+# Lending & Borrowing
 The lending & borrowing platform is a decentralized, non-custodial, pool-based interest rates platform that enables users to supply assets to earn interest, and to borrow against them to unlock liquidity, similar in nature to Aave, Compound, and other pool-based protocols.
 The platform runs natively on the NEAR blockchain, a layer 1, proof-of-stake, sharded blockchain with a WebAssembly runtime. The protocol's smart contracts are written in Rust.
 The protocol aims to unlock liquidity for interest-bearing assets, particularly layer 1 staking derivatives such as stNEAR and stETH. For instance, users will be able to deposit stNEAR as collateral, then borrow more NEAR to create a leveraged staking position, or borrow a stablecoin to create a self-repaying position.
-Access the app at:
-https://lending.rhea.finance/
-Previous
-Liquidity Pool Farming
-Next
-Supplying
-Last updated
-7 months ago
-Was this helpful?
-[[LINK TARGETS]]
-LINK | Markdown | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing.md
-LINK | Next Supplying | https://guide.rhea.finance/near-chain-guides/lending-and-borrowing/supplying
-LINK | Previous Liquidity Pool Farming | https://guide.rhea.finance/near-chain-guides/liquidity-pool-farming
-LINK | [no text] | https://docs.burrow.finance/product-docs/introduction/audits-and-risks
-LINK | https://lending.rhea.finance/ | https://lending.rhea.finance/
-LINK | llms.txt | https://guide.rhea.finance/llms.txt
+Access the app at: <https://lending.rhea.finance/>
+[<br>](https://docs.burrow.finance/product-docs/introduction/audits-and-risks)
