--- near/rhea-guide/x-chain-guides__liquidity-pools__6eb56beab1:previous
+++ near/rhea-guide/x-chain-guides__liquidity-pools__6eb56beab1:current
@@ -1,11 +1,7 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
-Last updated
-1 year ago
-Was this helpful?
-[[LINK TARGETS]]
-LINK | Markdown | https://guide.rhea.finance/x-chain-guides/liquidity-pools.md
-LINK | llms.txt | https://guide.rhea.finance/llms.txt
+> For the complete documentation index, see [llms.txt](https://guide.rhea.finance/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://guide.rhea.finance/x-chain-guides/liquidity-pools.md).
+# Liquidity Pools
+- [Adaptive Liquidity Market Maker (ALMM)](https://guide.rhea.finance/x-chain-guides/liquidity-pools/adaptive-liquidity-market-maker-almm.md)
+- [Concentrated Liquidity Market Maker (CLMM)](https://guide.rhea.finance/x-chain-guides/liquidity-pools/concentrated-liquidity-market-maker-clmm.md)
+- [Classic Pools](https://guide.rhea.finance/x-chain-guides/liquidity-pools/classic-pools.md): "Classic" Liquidity Pools
+- [Stable & Rated Pools](https://guide.rhea.finance/x-chain-guides/liquidity-pools/stable-and-rated-pools.md): Stable pools, which can contain two or three tokens, use Curve's StableSwap algorithm. Rated pools are for yield-bearing tokens, and are based on a variation of Curve's StableSwap algorithm.
+- [RHEA v2 Pools](https://guide.rhea.finance/x-chain-guides/liquidity-pools/rhea-v2-pools.md): Pooling with RHEA v2's Discretized Concentrated Liquidity AMM
