--- hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__hip-4-deployer-actions__edc41f3eaa:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__hip-4-deployer-actions__edc41f3eaa:current
@@ -316,9 +316,9 @@
 HIP-3 deployer actions
 Next
 HyperEVM
 Last updated
-10 days ago
+9 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/hip-4-deployer-actions.md
 LINK | Next HyperEVM | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/hyperevm
 LINK | Previous HIP-3 deployer actions | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/hip-3-deployer-actions
