--- hpl/hyperliquid-docs/hyperliquid-docs__onboarding__testnet-faucet__e9645bf7ef:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__onboarding__testnet-faucet__e9645bf7ef:current
@@ -14,9 +14,9 @@
 Export your email wallet
 Next
 HyperCore
 Last updated
-5 days ago
+6 days ago
 [[LINK TARGETS]]
 LINK | Export your email wallet | https://hyperliquid.gitbook.io/hyperliquid-docs/onboarding/export-your-email-wallet
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/onboarding/testnet-faucet.md
 LINK | Next HyperCore | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore
