--- hpl/hyperliquid-docs/hyperliquid-docs__trading__portfolio-graphs__33e2a56b28:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__trading__portfolio-graphs__33e2a56b28:current
@@ -0,0 +1,25 @@
+For the complete documentation index, see
+llms.txt
+. This page is also available as
+Markdown
+.
+The portfolio page shows account value and P&L graphs on 24 hour, 7 day, and 30 day time horizons.
+Account value
+includes unrealized pnl from cross and isolated margin positions, as well as vault balances.
+Pnl is defined as
+account value
+plus net deposits, i.e.
+account value + deposits - withdrawals
+.
+Note that these graphs are samples on deposits and withdrawals and also every 15 minutes. Therefore, they are not recommended to precise accounting purposes, as the interpolation between samples may not reflect the actual change in unrealized pnl in between two consecutive samples.
+Previous
+Market making
+Next
+Miscellaneous UI
+Last updated
+1 year ago
+[[LINK TARGETS]]
+LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/portfolio-graphs.md
+LINK | Next Miscellaneous UI | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/miscellaneous-ui
+LINK | Previous Market making | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/market-making
+LINK | llms.txt | https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt
