--- hpl/hyperliquid-docs/hyperliquid-docs__hypercore__aligned-quote-assets__ad90a91335:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__hypercore__aligned-quote-assets__ad90a91335:current
@@ -43,9 +43,9 @@
 Permissionless spot quote assets
 Next
 HyperEVM
 Last updated
-6 days ago
+5 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore/aligned-quote-assets.md
 LINK | Next HyperEVM | https://hyperliquid.gitbook.io/hyperliquid-docs/hyperevm
 LINK | Previous Permissionless spot quote assets | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore/permissionless-spot-quote-assets
