--- hpl/hyperliquid-docs/hyperliquid-docs__trading__hyperps__0309fe9a39:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__trading__hyperps__0309fe9a39:current
@@ -30,9 +30,9 @@
 Self-trade prevention
 Next
 Delisting
 Last updated
-5 days ago
+6 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/hyperps.md
 LINK | Next Delisting | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/delisting
 LINK | Previous Self-trade prevention | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/self-trade-prevention
