--- hpl/hyperliquid-docs/hyperliquid-docs__support__faq__unstaking-and-link-staking-issues__752bafdddc:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__support__faq__unstaking-and-link-staking-issues__752bafdddc:current
@@ -1,20 +1,8 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
-Unstaking transfer taking more than 7 days
-Staking and trading account linking issues
-Previous
-Portfolio margin
-Next
-Unstaking transfer taking more than 7 days
-Last updated
-1 year ago
-[[LINK TARGETS]]
-LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues.md
-LINK | Next Unstaking transfer taking more than 7 days | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues/unstaking-transfer-taking-more-than-7-days
-LINK | Previous Portfolio margin | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/portfolio-margin
-LINK | Staking and trading account linking issues | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues/staking-and-trading-account-linking-issues
-LINK | Unstaking transfer taking more than 7 days | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues/unstaking-transfer-taking-more-than-7-days
-LINK | llms.txt | https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt
+> For the complete documentation index, see [llms.txt](https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues.md).
+# Unstaking & link staking issues
+{% content-ref url="/pages/1JKaMuIWympI5a1oTvY2" %}
+[Unstaking transfer taking more than 7 days](/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues/unstaking-transfer-taking-more-than-7-days.md)
+{% endcontent-ref %}
+{% content-ref url="/pages/LZ4tWTmR4YmNKN6ksiZy" %}
+[Staking and trading account linking issues](/hyperliquid-docs/support/faq/unstaking-and-link-staking-issues/staking-and-trading-account-linking-issues.md)
+{% endcontent-ref %}
