--- hpl/hyperliquid-docs/hyperliquid-docs__hypercore__usdc__4fd3af5ec6:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__hypercore__usdc__4fd3af5ec6:current
@@ -16,9 +16,9 @@
 Overview
 Next
 API servers
 Last updated
-16 days ago
+15 days ago
 [[LINK TARGETS]]
 LINK | Audits | https://hyperliquid.gitbook.io/hyperliquid-docs/audits
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore/usdc.md
 LINK | Next API servers | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore/api-servers
