--- hpl/hyperliquid-docs/hyperliquid-docs__trading__uniswap-perpetuals__b91610779b:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__trading__uniswap-perpetuals__b91610779b:current
@@ -0,0 +1,15 @@
+For the complete documentation index, see
+llms.txt
+. This page is also available as
+Markdown
+.
+Some perpetual contracts on Hyperliquid use Uniswap V2 or V3 AMM price as the underlying spot asset. These contracts are restricted to be isolated-only, which means that cross margin is not allowed and margin cannot be manually removed from an open position. Instead, the position must be partially or fully closed to partially or fully return isolated margin.
+Uniswap pool prices are always converted to USDT prices based on the robust CEX oracle prices.
+The current contract addresses for the Uniswap pools used as oracles:
+RLB:
+0x510100d5143e011db24e2aa38abe85d73d5b2177
+Last updated
+1 year ago
+[[LINK TARGETS]]
+LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/uniswap-perpetuals.md
+LINK | llms.txt | https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt
