--- hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__deploying-hip-1-and-hip-2-assets__6ee18730fe:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__deploying-hip-1-and-hip-2-assets__6ee18730fe:current
@@ -75,9 +75,9 @@
 blacklistUsers?: Array<[string, boolean]>;
 }
 /**
 * Genesis denotes the initial creation of a token with a maximum supply.
-* @param maxSupply - Checksum ensureing all calls to UserGenesis succeeded
+* @param maxSupply - Checksum ensuring all calls to UserGenesis succeeded
 * @param noHyperliquidity - Set hyperliquidity balance to 0.
 */
 type Genesis = {
 token: number;
@@ -113,9 +113,9 @@
 * can be resent multiple times as long as the fee share is not increasing.
 * @param token - The token
 * @param share - The deployer trading fee share. Range: ["0%", "100%"]. Examples: "0.012%", "99.4%"
 */
-type SetDeployerTradingFeeShare {
+type SetDeployerTradingFeeShare = {
 token: number;
 share: string;
 }
 /**
