--- hpl/hyperliquid-docs/hyperliquid-docs__onboarding__export-your-email-wallet__631840b1ee:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__onboarding__export-your-email-wallet__631840b1ee:current
@@ -14,9 +14,9 @@
 Connect mobile via QR code
 Next
 Testnet faucet
 Last updated
-15 days ago
+16 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/onboarding/export-your-email-wallet.md
 LINK | Next Testnet faucet | https://hyperliquid.gitbook.io/hyperliquid-docs/onboarding/testnet-faucet
 LINK | Previous Connect mobile via QR code | https://hyperliquid.gitbook.io/hyperliquid-docs/onboarding/connect-mobile-via-qr-code
