--- hpl/hyperliquid-docs/hyperliquid-docs__trading__fees__032be05e4e:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__trading__fees__032be05e4e:current
@@ -363,9 +363,9 @@
 Trading
 Next
 Sub-accounts
 Last updated
-3 days ago
+4 days ago
 [[LINK TARGETS]]
 LINK | Aligned quote assets | https://hyperliquid.gitbook.io/hyperliquid-docs/hypercore/aligned-quote-assets
 LINK | HIP-3 deployer actions | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/hip-3-deployer-actions
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/trading/fees.md
