--- hpl/hyperliquid-docs/hyperliquid-docs__hyperliquid-improvement-proposals-hips__hip-3-builder-deployed-perpetuals__891bb7ea27:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__hyperliquid-improvement-proposals-hips__hip-3-builder-deployed-perpetuals__891bb7ea27:current
@@ -53,9 +53,9 @@
 HIP-2: Hyperliquidity
 Next
 HIP-4: Outcome markets
 Last updated
-17 days ago
+18 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/hyperliquid-improvement-proposals-hips/hip-3-builder-deployed-perpetuals.md
 LINK | Next HIP-4: Outcome markets | https://hyperliquid.gitbook.io/hyperliquid-docs/hyperliquid-improvement-proposals-hips/hip-4-outcome-markets
 LINK | Previous HIP-2: Hyperliquidity | https://hyperliquid.gitbook.io/hyperliquid-docs/hyperliquid-improvement-proposals-hips/hip-2-hyperliquidity
