--- hpl/hyperliquid-docs/hyperliquid-docs__support__faq__hyperevm-issues__0cd3461a0e:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__support__faq__hyperevm-issues__0cd3461a0e:current
@@ -1,22 +1,11 @@
-For the complete documentation index, see
-llms.txt
-. This page is also available as
-Markdown
-.
-Accidentally transferred to HyperEVM
-Can’t see my HyperEVM assets in wallet extension
-Gas problem on EVM
-Previous
-What does "Action already expired" mean?
-Next
-Accidentally transferred to HyperEVM
-Last updated
-1 year ago
-[[LINK TARGETS]]
-LINK | Accidentally transferred to HyperEVM | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues/accidentally-transferred-to-hyperevm
-LINK | Can’t see my HyperEVM assets in wallet extension | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues/cant-see-my-hyperevm-assets-in-wallet-extension
-LINK | Gas problem on EVM | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues/gas-problem-on-evm
-LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues.md
-LINK | Next Accidentally transferred to HyperEVM | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues/accidentally-transferred-to-hyperevm
-LINK | Previous What does "Action already expired" mean? | https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/trade-outcome-looks-incorrect/what-does-action-already-expired-mean
-LINK | llms.txt | https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt
+> For the complete documentation index, see [llms.txt](https://hyperliquid.gitbook.io/hyperliquid-docs/llms.txt). Markdown versions of documentation pages are available by appending `.md` to page URLs; this page is available as [Markdown](https://hyperliquid.gitbook.io/hyperliquid-docs/support/faq/hyperevm-issues.md).
+# HyperEVM issues
+{% content-ref url="/pages/yM5Wdf4MOmf8G4Ru3bX0" %}
+[Accidentally transferred to HyperEVM](/hyperliquid-docs/support/faq/hyperevm-issues/accidentally-transferred-to-hyperevm.md)
+{% endcontent-ref %}
+{% content-ref url="/pages/hmUUBqR6JG7mmzD4agBW" %}
+[Can’t see my HyperEVM assets in wallet extension](/hyperliquid-docs/support/faq/hyperevm-issues/cant-see-my-hyperevm-assets-in-wallet-extension.md)
+{% endcontent-ref %}
+{% content-ref url="/pages/EUjqAfnOiHdmyS9Pg1vZ" %}
+[Gas problem on EVM](/hyperliquid-docs/support/faq/hyperevm-issues/gas-problem-on-evm.md)
+{% endcontent-ref %}
