--- hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__hip-3-deployer-actions__01aa5df09f:previous
+++ hpl/hyperliquid-docs/hyperliquid-docs__for-developers__api__hip-3-deployer-actions__01aa5df09f:current
@@ -278,9 +278,9 @@
 Deploying HIP-1 and HIP-2 assets
 Next
 HIP-4 deployer actions
 Last updated
-17 days ago
+18 days ago
 [[LINK TARGETS]]
 LINK | Markdown | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/hip-3-deployer-actions.md
 LINK | Next HIP-4 deployer actions | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/hip-4-deployer-actions
 LINK | Previous Deploying HIP-1 and HIP-2 assets | https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/deploying-hip-1-and-hip-2-assets
