--- hpl/hyperlend-docs/developer-documentation__isolated-pools__liquidations__1731955029:previous
+++ hpl/hyperlend-docs/developer-documentation__isolated-pools__liquidations__1731955029:current
@@ -0,0 +1,23 @@
+For the complete documentation index, see
+llms.txt
+. This page is also available as
+Markdown
+.
+Liquidations
+If a borrower’s LTV rises above the maximum limit, any user can liquidate the position by repaying the loan. In return, the liquidator receives collateral, along with a liquidation fee.
+The liquidation fee is different if the liquidator repays the entire position (
+cleanLiquidationFee
+) or only a portion (
+dirtyLiquidationFee
+, 90% of the clean fee).
+Previous
+↪ key-concepts
+Next
+↪ interest rates
+Last updated
+4 months ago
+[[LINK TARGETS]]
+LINK | Markdown | https://docs.hyperlend.finance/developer-documentation/isolated-pools/liquidations.md
+LINK | Next ↪ interest rates | https://docs.hyperlend.finance/developer-documentation/isolated-pools/interest-rates
+LINK | Previous ↪ key-concepts | https://docs.hyperlend.finance/developer-documentation/isolated-pools/key-concepts
+LINK | llms.txt | https://docs.hyperlend.finance/llms.txt
